circe/decoupled_pricing/compute.py

```diff
@@ -320,67 +322,86 @@ def execute_task(home_id,task_name,file_name, filenames, input_path, output_path
     dag_task.join()
     
     
-def transfer_mapping_decorator(TRANSFER=0):
-    """Mapping the chosen TA2 module (network and resource monitor) based on ``jupiter_config.PROFILER`` in ``jupiter_config.ini``
+
+def transfer_data_scp(ID,user,pword,source, destination):
```

Comment:
The reason why we want to split changes into smaller commits is to then be able to understand what you are doing. Changing print() to logging() should be one commit so i can easily just approve that. That will then leave me to better understand what you are trying to do here with transfer_data_scp().

Reply:
This is the TA1/TA2 interface. I made it easier to use. Not changed much.